src/tomography/tomographyutils.jl

```diff
@@ -0,0 +1,137 @@
+function trace_outputsites(L::LPDO)
```

Comment:
Without looking too carefully at this, could this have an interface like `tr(L::LPDO, tag::String)` where `tr(L, "Output")` traces the output sites and `tr(L, "Input")` traces the input sites? We could stick with the more verbose function name for now and switch the that more generic version in the future.

Reply:
Sounds good, let's do it in some next PR